install/kubernetes/cilium/templates/cilium-nodeinit-daemonset.yaml

```diff
@@ -242,6 +251,15 @@ spec:
                   docker kill "${container_id}" || true
                 done
               else
+                # Wait until cilium-agent is up and running which means the cni
```

Comment:
FWIW locally here's an example output I expect from `crictl ps`, might be handy to have in-line so we can reason about the logic later on:

```
# crictl -r /var/snap/microk8s/common/run/containerd.sock ps | grep cilium-agent
e0bb0049e69ad       70cba308a65dc       2 hours ago         Running             cilium-agent           4                   d676816b2fa21
```